bridgestyle/arcgis/togeostyler.py

```diff
@@ -127,6 +130,85 @@ def processClassBreaksRenderer(renderer, options):
     return rules
 
 
+def processChartRenderer(renderer, options):
+    rules = []
+
+    # Basic setup for chart renderer
+    to_lower = options.get("tolowercase", False)
+    fields = renderer.get("fieldNames", [])
+    norm_fields = [f.lower() if to_lower else f for f in fields]
+
+    rotation = _getSymbolRotationFromVisualVariables(renderer, to_lower)
+
+    symbol_layers = renderer.get("chartSymbol", {}).get("symbol", {}).get("symbolLayers", [])
+    chart_layer = next(
+        (
+            ly
+            for ly in symbol_layers
+            if ly.get("type")
+            in {"CIMPieChartMarker", "CIMBarChartMarker", "CIMStackedBarChartMarker"}
+        ),
+        None
+    )
+    if not chart_layer:
+        return rules # unsupported chart layer
```

Comment:
Could you please add something here so a user knows when an unsupported symbol layer was encountered? So by adding `_warnings.append("...")`?